packages/platforms/react-native/rollup.config.mjs

```diff
@@ -20,7 +20,6 @@ function reactNativeSpecPlugin () {
     buildStart () { 
       this.emitFile({
         type: 'prebuilt-chunk',
-        id: 'lib/NativeBugsnagPerformance.ts',
```

Comment:
`id` isn't valid for a `prebuilt-chunk`: https://rollupjs.org/plugin-development/#this-emitfile